### PR TITLE
Data file for testing backwards compatibility of container serialization

### PR DESCRIPTION
**Pull request by coclauso**
See discussion on related PR:
https://github.com/microsoft/FluidFramework/pull/6218

### CODE REVIEW

**Review by anthony-murphy — APPROVED** [review]

**Comment by coclauso** [comment]
Thanks for the review, @anthony-murphy .  It looks like I can't merge this request, since I don't have write access to the repo.

**Pull request merged**
